src/openrct2/ride/Vehicle.h

```diff
@@ -427,17 +459,17 @@ enum : uint32_t
     VEHICLE_ENTRY_FLAG_ALLOW_DOORS_DEPRECATED = 1 << 8, // Not used any more - every vehicle will now work with doors.
     VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_2 = 1 << 9,
     VEHICLE_ENTRY_FLAG_RECALCULATE_SPRITE_BOUNDS = 1 << 10, // Only used during loading of the objects.
-    VEHICLE_ENTRY_FLAG_USE_16_ROTATION_FRAMES = 1
-        << 11, // Instead of the default 32 rotation frames. Only used for boat hire and works only for non sloped sprites.
```

Comment:
I don't know why clang format wants to move this around, but I don't argue because it's clang format

Reply:
Because of the line length limit. The comment is pretty long.